datanode/sqlstore/migrations/0001_initial.sql

```diff
@@ -1324,8 +1359,11 @@ DROP TYPE IF EXISTS proposal_error;
 DROP TYPE IF EXISTS proposal_state;
 
 DROP TABLE IF EXISTS epochs;
-DROP VIEW IF EXISTS delegations_current;
+DROP TABLE IF EXISTS delegations_current;
 DROP TABLE IF EXISTS delegations;
+DROP TRIGGER IF EXISTS update_current_delegations ON delegations;
+DROP FUNCTION IF EXISTS update_current_delegations;
```

Comment:
Probably not an issue, but could there be problem when dropping here, as we drop first the delegations_current, but it's used in the update_current_delegations function?

Reply:
don't think it does, but shifted it around anyways